.github/workflows/quality.yml

```diff
@@ -30,7 +30,10 @@ jobs:
         uses: ./.github/composite-actions/install
 
       - name: Run tests
-        run: pnpm test
+        uses: ArtiomTr/jest-coverage-report-action@v2
+        with:
+          package-manager: pnpm
+          test-script: pnpm test:ci
```

Comment:
Try fix it.

`npx jest --coverage --silent --testLocationInResults --ci --json --outputFile=report.json`

Reply:
See this PR.

https://github.com/hirotomoyamada/yamada-ui/pull/683

Reply:
I would like to try various GitHub Actions, so I'll merge this PR for now.